src/Resources/config/services.yml

```diff
@@ -0,0 +1,79 @@
+services:
+  # Decorate nelmio property accessor so it uses reflection
+  aligent.fixtures.reflection_property_accessor:
+    class: Nelmio\Alice\PropertyAccess\ReflectionPropertyAccessor
+    public: false
+    decorates: nelmio_alice.property_accessor
+    decoration_priority: -10
+    arguments:
+      - '@aligent.fixtures.reflection_property_accessor.inner'
+
+  # Override Persister with Oro Testframework persister
+  fidry_alice_data_fixtures.persistence.persister.doctrine.object_manager_persister:
+    class: Oro\Bundle\TestFrameworkBundle\Test\DataFixtures\Persister\ObjectManagerPersister
+    lazy: true
+    arguments:
+      - '@doctrine.orm.entity_manager'
+
+  Aligent\FixturesBundle\Initializer\ReferenceInitializer:
```

Comment:
There's quite a few Initializers, so it might be worth moving them out into their own `initializers.yml` eventually. Sorting them alphabetically could also make the list easier to manage. Not super important though.